plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -433,12 +414,6 @@ function CoverMenu:onHistoryMenuHold(item)
     UIManager:close(self.histfile_dialog)
     UIManager:clearRenderStack()
 
-    -- Replace Book information callback to use directly our bookinfo
-    orig_buttons[2][2].callback = function()
-        FileManagerBookInfo:show(file, bookinfo)
-        UIManager:close(self.histfile_dialog)
-    end
-
```

Comment:
Moved this down after "Create the new ButtonDialog, and let UIManager show it" below, otherwise this newly overwritten callback is reset when that block runs